### ISSUE TITLE
Can we share this doc?!

### ISSUE THREAD

**Issue by Tarsio25** [mmmonteiro/designethics#52]
Hello!!
hope to find you all well!!!

Can we share this document?! The text/info I mean?! If so to whom should I give credit?! 
Mike Monteiro and anyone else?!

Thx! Best,
Rui

**Comment by jwishnie** [comment]
Yes please! The Code is licensed under CC BY-SA 4.0 . I've added a LICENSE.TXT file to clarify this

**Comment by Tarsio25** [comment]
Cool!! Thx Jeff!!!
Is there any editorial design file or can I make one?!

I thought this information was going to be part of some document designed
by Mule Design, or even a simple website... and we we're just contributing
with the translations

Thx!
Happy Thursday!!

**Comment by jwishnie** [comment]
Hi Rui,
You are welcome to use the ethics anyway you want including creating derivative works (like posters, books, etc…)

Mike and Scout Books made a little red book version: https://scoutbooks.com/design-ethics-mike-monteiro/
On Thu, Apr 19, 2018 at 10:10 AM, Rui Dias <notifications@github.com> wrote:
Cool!! Thx Jeff!!!
Is there any editorial design file or can I make one?!

I thought this information was going to be part of some document designed
by Mule Design, or even a simple website... and we we're just contributing
with the translations

Thx!
Happy Thursday!!

**Comment by Tarsio25** [comment]
Alright!!! :)
i will follow the license guidelines!!!

Keep on smilin'! - Bob Marley
